src/routingtable.go

```diff
@@ -68,3 +85,12 @@ func (routingTable *RoutingTable) getBucketIndex(id *KademliaID) int {
 
 	return IDLength*8 - 1
 }
+
+
+func (routingTable *RoutingTable) GetMe() Contact {
+	routingTable.mutex.Lock()
+	defer routingTable.mutex.Unlock()
+
+	return routingTable.me
+}
```

Comment:
Is mutex locking really necessary here? routingTable.me should be constant throughout the entire program right?

Reply:
You are correct, It is not necessary. I added solely so that we can say that we use mutex and thus it is thread safe to the teacher when we have our sprint review.